config.ru

```diff
@@ -21,7 +21,9 @@ end
 if Builds::BUILD_CONFIG['teamCityBaseUrl']
   require 'teamcity'
   TeamCity.configure do |config|
-    config.endpoint = Builds::BUILD_CONFIG['teamCityBaseUrl'] + '/app/rest?guest=1'
+    config.endpoint = Builds::BUILD_CONFIG['teamCityBaseUrl'] + '/app/rest'
```

Comment:
I think that removing the `guest` query string will likely break existing configurations for servers that don't require authentication.

Reply:
Ok do you have any suggestion how to allow both guest and user. Our company does only have authentication without guest login.

Reply:
fixed